public/app/themes/justice/src/sass/admin.scss

```diff
@@ -1,3 +1,6 @@
+@import "justice-theme";
+@import "admin-bar-logo";
+
```

Comment:
Small nit-pick, these will be easier to manage in the long term if they're in alphabetic order.